test.html

```diff
@@ -0,0 +1,50 @@
+<div style="background-image: url(&quot;https://static.wqxr.org/assets/img/backgrounds/streambanner-6ec7bbc84c9f49cc148318e3f8d25cbb.jpg&quot;);" class="stream-banner-card">
```

Comment:
what is this file for? if it's just for testing and you're done with it, you can delete it...